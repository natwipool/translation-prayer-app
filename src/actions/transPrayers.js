// SET_TRANS_PRAYER
const setTransPrayer = (transPrayers = {}) => ({
  type: 'SET_TRANS_PRAYER',
  transPrayers
});

export default setTransPrayer;