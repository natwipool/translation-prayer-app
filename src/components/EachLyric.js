import React from 'react';

const EachLyric = (props) => (
  <div>
    {props.lyric}
  </div>
);

export default EachLyric;