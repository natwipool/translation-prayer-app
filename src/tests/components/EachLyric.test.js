import React from 'react';
import { shallow } from 'enzyme';
import EachLyric from '../../components/EachLyric';
import transPrayers from '../fixtures/transPrayers';

test('should render EachLyric correctly', () => {
  const wrapper = shallow(<EachLyric lyric={transPrayers[2].lyrics[0]}/>);
  expect(wrapper).toMatchSnapshot();
});