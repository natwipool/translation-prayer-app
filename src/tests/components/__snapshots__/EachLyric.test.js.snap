// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render EachLyric correctly 1`] = `
<div>
  (นำ) หันทะ มะยัง สัพพะปัตติทานะคาถาโย ภะณามะ เส.
</div>
`;
